src/tests/functions.at

```diff
@@ -95,7 +95,9 @@ m4_define([FWD_RESTART], [
 ])
 
 m4_define([FWD_START_TEST], [
-    AT_SETUP([$1])
+    AT_SETUP(m4_if(m4_eval(m4_len([$1]) + m4_len(__file__) + m4_len(__line__) < 45), [1],
+          [__file__:__line__> $1],
+          [__file__:__line__> m4_substr([$1], 0, m4_eval(42 - m4_len(__file__) - m4_len(__line__)))… ]))
```

Comment:
this should be
```
[__file__:__line__> m4_substr([$1], 0, m4_eval(43 - m4_len(__file__) - m4_len(__line__)))…]))
                                               ^^                                        ^^
```
but that doesn't work. Suggestion welcome.

Reply:
🤷

Reply:
ok, then we live with the fact that when the name gets abbreviated, it gets abbreviated one character shorter than would be necessary... *sigh*